Resources/Documentation/linux_installation.md

```diff
@@ -50,7 +50,6 @@ The following instructions aim to minimize use of linuxbrew installs for package
  test -r ~/.bash_profile && echo "eval \$($(brew --prefix)/bin/brew shellenv)" >> ~/.bashrc
  ~~~
 
-  - Alternately some issues (such as the sdl2 conflict) have been avoided by adding Linuxbrew lower down in the path order than the Linuxbrew instructions call for (Such as by editing `/etc/environment` to include `/home/linuxbrew/.linuxbrew/bin` after the other $PATH directories.) Your mileage may vary!
```

Comment:
How come this was deleted outright rather than changed to say `Homebrew` like the other sections?

Reply:
Was the SDL2 issue not resolved? I cannot reproduce the error on any of the various different Linux configurations we have here (Debian 11.6, 10.13 and 9.13; Ubuntu 22.10, 22.04.1 LTS, 20.04.5 LTS and 18.04.6 LTS). My intent is to remove obsolete information, but of course if it is not obsolete it should remain.

Reply:
I'm actually not sure if it is no longer a problem or not! I don't seem to have problems with it on my systems, but I'm unsure how it would behave at this point in a totally fresh/clean install environment

Reply:
We do still have the section farther down about manually removing the brew version of sdl still in the [install directions](https://github.com/amiaopensource/vrecord/blob/1d54fa5f216385737d30d971d9380338d6988d84/Resources/Documentation/linux_installation.md#fix-conflicting-sdl2-dependencies)

Reply:
I reverted this part.